src/smdba/postgresqlgate.py

```diff
@@ -820,6 +829,8 @@ def _perform_enable_backups(self, **args: str) -> None:
 
             if os.path.exists("{0}/base.tar.gz".format(b_dir_temp)):
                 os.rename("{0}/base.tar.gz".format(b_dir_temp), "{0}/base.tar.gz".format(backup_dir))
+            if os.path.exists("{0}/backup_manifest".format(b_dir_temp)):
```

Comment:
You could also use `os.path.join` here. But this would also work. It's just not as nice.

Reply:
I just copied what was one line above ;)

Reply:
I am sure it would be better to have os.path.join, but then we would also need to change, in my opinion, the line above it, or it is weird to use different ways for the same thing (weird=makes it difficult to understand/read). I just found it weird to refactor existing/working code, so this is why I would prefer to leave it as it is. Sounds good?

Reply:
Sure, that's okay with me.